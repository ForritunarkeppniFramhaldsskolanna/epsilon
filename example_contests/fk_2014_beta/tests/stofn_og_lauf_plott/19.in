172
39 35 34 42 81 75 98 35 30 17 95 84 68 98 81 61 71 58 82 81 63 92 13 16 17 47 13 83 44 25 11 79 47 46 64 40 92 66 33 87 59 98 11 20 47 84 13 29 79 60 48 86 52 40 34 49 33 13 44 87 68 42 91 90 48 40 67 94 18 66 52 69 93 46 44 52 27 70 65 35 23 77 86 51 15 41 23 94 23 66 65 89 80 64 70 72 76 45 47 89 51 43 33 54 72 57 18 47 70 73 66 77 53 74 60 70 79 12 45 61 48 14 15 36 90 57 86 35 68 30 24 34 98 88 20 43 32 56 97 28 31 24 58 69 90 85 71 85 73 34 68 42 45 48 81 21 97 43 48 15 79 35 55 86 36 77 85 58 46 32 84 74
