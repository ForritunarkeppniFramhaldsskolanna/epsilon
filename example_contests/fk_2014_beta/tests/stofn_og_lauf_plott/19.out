1 1123333455567788
2 001333445789
3 00122333444455555669
4 000122233344455566677777888889
5 112223456778889
6 00113445566667888899
7 000011223344567779999
8 0111123444555666677899
9 0001223445778888
